README.md

```diff
@@ -372,3 +372,12 @@ ahead to the failed step and try to rerun it.
     (https://cloud.google.com/genomics/v1alpha2/pipelines).
 *   Read about [Dataflow](https://cloud.google.com/dataflow).
 *   Write your own workflows!
+
+## A word about quotas
```

Comment:
FAQ and troubleshooting

### What if I want to run a large batch of parallel jobs?

Google Cloud Platform has various quotas ...